index.html

```diff
@@ -1,12 +1,15 @@
-
+<!DOCTYPE html>
 <!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
 <!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
 <!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
-<!--[if gt IE 8]><!--> <html class="no-js"> <!--<![endif]-->
+<!--[if gt IE 8]><!-->
+<html class="no-js">
+<!--<![endif]-->
+
 <head>
     <meta charset="utf-8">
-    <title>Eye</title>
-    <link href='http://fonts.googleapis.com/css?family=Nova+Round' rel='stylesheet' type='text/css'>
+    <title>Cells, Tissues &amp; Organs</title>
```

Comment:
it is okay to use specials chars like `&` since you added the charset meta

Reply:
Thanks for the comment. Did you mean that if we use charset then we don't have to use this ampersand after the special character &?